website/docs/en/api/javascript-api/index.mdx

```diff
@@ -288,3 +288,65 @@ const source = new RawSource('console.log("Hello, world!");');
 ```
 
 For detailed usage, please refer to the [webpack-sources](https://github.com/webpack/webpack-sources) documentation.
+
+## javascript minify and transform
```

Comment:
Consider that we may introduce more SWC APIs or examples in the future, can we add a separate "SWC API" page under the "JavaScript API"?

![image](https://github.com/user-attachments/assets/574bf987-0119-46fa-a046-945c15a59189)

Reply:
Maybe we can stay here in temporary.
We can move it to a separate parts that named `SWC API`, if we have more SWC APIs, 
Currently, we only provide the export of the transform and minify APIs. The separate part seems a bit monotonous.

Reply:
Actually we can add more content and examples to this page:

- Importing SWC types from `@rspack/core`
- Using SWC API in a loader
- Using SWC API in a plugin

Reply:
> Actually we can add more content and examples to this page:
> 
> * Importing SWC types from `@rspack/core`
> * Using SWC API in a loader
> * Using SWC API in a plugin

Actually, I don't want to export it to all users so fast, Because it's a experiments api. This api exports is to satisfy `rslib`, `rsbuild` require.
I think it would be suitable to add more content once those APIs become stable.

Reply:
We should mention in the documentation that this is an experimental API and the API signature may change in non-major releases.

Also, adding examples for experimental APIs is not a bad thing, because experimental APIs are usually brand new and adding some examples can help the first users try to understand and try it.

Reply:
Done~